src/ring_theory/ideal/basic.lean

```diff
@@ -77,6 +77,9 @@ begin
   rwa [← mul_assoc, hy', one_mul] at this,
 end
 
+lemma ne_bot_of_ne_bot_le {I J : ideal α} (h : I ≤ J) (hI : I ≠ ⊥) : J ≠ ⊥ :=
+ ne_of_gt (lt_of_lt_of_le (bot_lt_iff_ne_bot.2 hI) h)
```

Comment:
This doesn't need anything specific to ideals: the proof applies to any partial order with a bottom element.

I also applied a few abbreviations to the proof, which is not so important and you can ignore.
```suggestion
-- can be moved to order/bounded_lattice.lean, after `ne_bot_of_gt`
lemma ne_bot_of_ne_bot_le {i j : α} [order_bot α] (h : i ≤ j) (hi : i ≠ ⊥) : j ≠ ⊥ :=
((bot_lt_iff_ne_bot.2 hi).trans_le h).ne'
```

Reply:
Oh, well I just found the exact same lemma in `order/bounded_lattice.lean`, it seems someone got there (3 years) before us ;-)